content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
 
-.. image:: reporting/favorites.png
-   :align: center
-   :alt: Favorite options for reports.
+.. example::
+   The :guilabel:`Work Entries Analysis` report has two default filters, the :guilabel:`Current
+   month:(Month)(Year)` filter, and the :guilabel:`Validated` filter. The :guilabel:`Payroll` report
+   only has one default filter, the :guilabel:`Last 365 Days Payslip` filter.
+
+   .. image:: reporting/custom-filter.png
+      :align: center
+      :alt: Filters enabled for the Work Entries Analysis report.
+
+All reports can include custom filters or group information by different metrics (employee,
+department, company, etc.). Some reports have the option to compare the current report to the
+previous time period or year (a :guilabel:`Comparison` option).
+
+Click on a parameter to select it and activate it. The report is immediately updated with the new
+parameters.
+
+The updated report can be set as a favorite report, meaning the parameters are stored for quick
+access in the future. First, click :guilabel:`Save the current search` under the
+:guilabel:`Favorites` section, and two options and a :guilabel:`Save` button appear.
+
+To set the current report as the default configuration when the report is accessed, check the box
+next to :guilabel:`Default filter`. If the current report should be accessible to everyone in the
+database, check the box next to :guilabel:`Share`.
+
+Finally, click :guilabel:`Save`, which saves the currently configured report, and it now appears
+beneath :guilabel:`Favorites`.
+
+Default reports
+===============
 
 Payroll
 -------
 
-Click on :menuselection:`Payroll --> Reporting --> Payroll` to display the :guilabel:`Payroll
-Analysis` report.
+Click on :menuselection:`Payroll app --> Reporting --> Payroll` to display the :guilabel:`Payroll
+Analysis` report. This report shows all the payslips generated in the last 365 days.
 
 .. image:: reporting/payroll-report.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Payroll overview report showing payroll for the last 365 days.
+
+The report can display metrics for a variety of parameters. Click the :guilabel:`Measures` box to
+view a drop-down menu with the various options to display.
 
-This shows all the payslips generated in the last 365 days. The report can display metrics for a
-variety of parameters. Click the :guilabel:`Measures` box to view a drop-down menu with the various
-options to display. The options available include: :guilabel:`# Payslip`, :guilabel:`Basic Wage`,
+The default options available include: :guilabel:`# Payslip`, :guilabel:`Basic Wage`,
 :guilabel:`Basic Wage for Time Off`, :guilabel:`Days of Paid Time Off`, :guilabel:`Days of
 Unforeseen Absence`, :guilabel:`Days of Unpaid Time Off`, :guilabel:`Gross Wage`, :guilabel:`Net
 Wage`, :guilabel:`Number of Days`, :guilabel:`Number of Hours`, :guilabel:`Work Days`,
 :guilabel:`Work Hours`, and :guilabel:`Count`.
```

Comment:
For improved appearance and easier readability, this may be better served as a list